### PR TITLE
Test to verify hwinfo tool options - V3

### PR DESCRIPTION
**Pull request by Pavithra1602**
hwinfo is used to probe for the hardware present in the system.

Signed-off-by: pavrampu@linux.vnet.ibm.com <pavrampu@in.ibm.com>

### CODE REVIEW

File: generic/hwinfo.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            self.skip("supported only on Power platform")
+        if dist.name == 'redhat':
+            self.skip('Hwinfo not supported on RHEL')
+        sm = SoftwareManager()
+        if not sm.check_installed("hwinfo") and not sm.install("hwinfo"):
+            self.error("Fail to install hwinfo required for this test.")
+
+    def test(self):
+        self.log.info("===============Executing hwinfo tool test===============")
+        list = ['--all', '--arch', '--bios', '--block', '--bluetooth', '--braille',
```
Comment (PraveenPenguin) [inline generic/hwinfo.py]: can this be pass via yaml file as if we want to test only single entry ?

Comment (PraveenPenguin) [inline generic/hwinfo.py]: it just a suggestion as this is your test :) if you want to like this than ok

File: generic/hwinfo.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                '--sound', '--storage-ctrl', '--sys', '--tape', '--tv', '--uml', '--usb'
+                '--wlan', '--xen', '--zip', '--short', '--listmd']
+        for list_item in list:
+            cmd = "hwinfo %s" % list_item
+            self.run_cmd(cmd)
+        disk_name = process.system_output("df -h | egrep '(s|v)d[a-z][1-8]' | "
+                                          "tail -1 | cut -d' ' -f1",
+                                          shell=True).strip("12345")
+        self.run_cmd("hwinfo --disk --only %s" % disk_name)
+        Unique_Id = process.system_output("hwinfo --disk --only /dev/sdb | "
```
Comment (nasastry) [inline generic/hwinfo.py]: `/dev/sdb` is always available on your machine? what if it is not available?

**Comment by Pavithra1602** [comment]
Raised new PR.
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/346

**Pull request closed without merging**